packages/serverpod/lib/src/server/websocket_request_handlers/method_websocket_request_handler.dart

```diff
@@ -36,7 +36,13 @@ class MethodWebsocketRequestHandler {
         }
 
         switch (message) {
-          case OpenMethodStreamCommand():
+          case OpenMethodStreamCommand(
+              endpoint: var endpoint,
+              method: var method,
+              connectionId: var connectionId,
+            ):
```

Comment:
Question(For my knowledge): This was neat; I'm not that familiar with this. Does this actually create a new instance of `OpenMethodStreamCommand` or does it just assign a variable to the fields from the message received?

Reply:
It destructures the object being matched, i.e. does not create a new object, just assigns the specified vars from public properties of the matched object.

Actually the previous code was already using destructuring, we just didn't extract any attributes!